PocketKit/Sources/Textile/Views/Tags/TagsList.swift

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+public protocol TagsList {
+    func arrangeTags(with tags: [String]) -> [TagType]
+}
+
+public extension TagsList {
+    /// Arranges the tags in a list by sorting with three recent tags and then alphabetizing the rest
+    /// - Parameter tags: list of users tags
+    /// - Returns: a list of tags to display to the user
+    func arrangeTags(with tags: [String]) -> [TagType] {
```

Comment:
Is it possible that we try to arrange tag strings that can't be rendered into TagTypes

Reply:
The more I think about it, the more I believe that this is backend work.
We can't experiment with tag orders or the like with this code.

Reply:
> Is it possible that we try to arrange tag strings that can't be rendered into TagTypes
Tag Types can take in a String, so not sure what care it would be not possible.

> The more I think about it, the more I believe that this is backend work.
Interesting point, I agree that it will be nice to have it backend. Post MVP - recent tags will be a backend call, but as for now we are using recent tags created locally. 

> We can't experiment with tag orders or the like with this code.
Can you expand on this? thanks!